auto_nag/scripts/regression.py

```diff
@@ -40,24 +40,13 @@ def get_bz_params(self, date):
         reporter_blacklist = ','.join(reporter_blacklist)
 
         params = {
-            'f1': 'keywords',
-            'o1': 'nowords',
-            'v1': 'regression,feature,meta',
-            'f2': 'longdesc',
-            'o2': 'anywordssubstr',
-            'v2': 'regress caus',
-            'f3': 'resolution',
-            'o3': 'nowords',
-            'v3': resolution_blacklist,
-            'f4': 'longdesc',
-            'o4': 'changedafter',
-            'v4': start_date,
-            'f5': 'longdesc',
-            'o5': 'changedbefore',
-            'v5': end_date,
-            'f6': 'reporter',
-            'o6': 'nowords',
-            'v6': reporter_blacklist,
+            'f1': 'keywords', 'o1': 'nowords', 'v1': 'regression,feature,meta',
+            'f2': 'longdesc', 'o2': 'anywordssubstr', 'v2': 'regress caus',
+            'f3': 'resolution', 'o3': 'nowords', 'v3': resolution_blacklist,
+            'f4': 'longdesc', 'o4': 'changedafter', 'v4': start_date,
+            'f5': 'longdesc', 'o5': 'changedbefore', 'v5': end_date,
+            'f6': 'reporter', 'o6': 'nowords', 'v6': reporter_blacklist,
+            'bug_type': 'defect',
```

Comment:
nit: just add an entry "types": ["defect"] in tools.json for regression tool

Reply:
Done with e684bd4.